flambe/experiment/experiment.py

```diff
@@ -194,11 +195,8 @@ def run(self, force: bool = False, verbose: bool = False, debug: bool = False, *
 
         # Check if save_path/name already exists + is not empty
         # + force and resume are False
-        if (
-            os.path.exists(self.full_save_path) and
-            os.listdir(self.full_save_path) and
-            not self.resume and not force
-        ):
+        if not self.resume and not force and os.path.exists(self.full_save_path) \
+                and list(get_files(self.full_save_path)):
```

Comment:
use `os.listdir` instead

Reply:
`os.listdir` is non recursive and in remote clusters there is an empty folder created in the output folder (called `_resources`), so `os.listdir` resolves to `True` in this conditional... causing the bug I describe above.

This fix uses a recursive implementation where it checks if any file exists. So if a remote execution is taking place, then even if there is a empty folder, this will resolve to `False`.

Reply:
how about `os.walk`?

Reply:
or `glob` maybe?

Reply:
`get_files` uses `os.walk` internally @jeremyasapp